cmake/coverage.cmake

```diff
@@ -84,6 +84,11 @@ ie_coverage_extract(INPUT "openvino" OUTPUT "core"
 ie_coverage_genhtml(INFO_FILE "core"
                     PREFIX "${OV_COVERAGE_BASE_DIRECTORY}")
 
+ie_coverage_extract(INPUT "openvino" OUTPUT "openvino_all"
```

Comment:
BTW why this code doesn't work:
https://github.com/openvinotoolkit/openvino/pull/12655/files#diff-b12da2a03216a92775c95e1a1ea9cf68c5d1f937cdcf1a5419ca7249895a67ddR7-R11

? 
I think we can fix patterns and use openvino name for the common report 😄